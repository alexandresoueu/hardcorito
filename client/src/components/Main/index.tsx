const Main = () => (
  <main>
    <h1>Bulbassauros</h1>
  </main>
)

export default Main
