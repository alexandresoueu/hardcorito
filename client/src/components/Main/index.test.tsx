import { render, screen } from '@testing-library/react'

import Main from '.'

describe('MAIN', () => {
  it('mainzin', () => {
    const { container } = render(<Main />)

    expect(
      screen.getByRole('heading', { name: /bulbassauro/i })
    ).toBeInTheDocument()

    expect(container.firstChild).toMatchSnapshot()
  })
})
