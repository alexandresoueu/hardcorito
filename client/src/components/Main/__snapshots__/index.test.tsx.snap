// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`MAIN mainzin 1`] = `
<main>
  <h1>
    Bulbassauro
  </h1>
</main>
`;
